mesos-swarm-marathon/agent-gt0.json

```diff
@@ -123,7 +135,7 @@
     "storageAccountsCount": "[add(div(parameters('agentCount'), variables('vmsPerStorageAccount')), mod(add(mod(parameters('agentCount'), variables('vmsPerStorageAccount')),2), add(mod(parameters('agentCount'), variables('vmsPerStorageAccount')),1)))]",
     "storageAccountType": "Standard_GRS",
     "wgetCommandPrefix": "[concat('wget --tries 20 --retry-connrefused --waitretry=15 -qO- ', parameters('customScriptLocation'), 'configure-mesos-cluster.sh | nohup /bin/bash -s ')]",
-    "wgetCommandPostfix":  " > /var/log/azure/firstinstall.log 2>&1 &'",
+    "wgetCommandPostfix":  " >> /var/log/azure/cluster-bootstrap.log 2>&1 &'",
```

Comment:
Must be my Canadian background . . . From Bing: In linguistics, a suffix (also sometimes termed postfix or ending or, in older literature, affix) is an affix which is placed after the stem of a word